packages/math-input/src/index.ts

```diff
@@ -7,7 +7,9 @@ import "../less/main.less";
 export {CursorContext} from "./components/input/cursor-contexts";
 export {default as KeypadInput} from "./components/input/math-input";
 export {keypadElementPropType} from "./components/prop-types";
-export {default as Keypad} from "./components/provided-keypad";
+export {default as ProvidedKeypad} from "./components/provided-keypad";
 export {default as KeyConfigs} from "./data/key-configs";
 export {default as Keys} from "./data/keys";
 export {KeyType, KeypadType} from "./enums";
+
+export {default as Keypad} from "./components/keypad/index";
```

Comment:
I'm not opposed to this, but we'll need to be careful as we roll out this change to webapp/mobile.  I'm tempted to call the v1 keypad `KeypadV1` or even `LegacyKeypad` to make it clear where we still use it that it's on the way out. 🤔  You decide. :)